src/ui/components/Library/Team/View/TestRuns/TestRunListItem.tsx

```diff
@@ -17,7 +17,7 @@ function Status({ failCount }: { failCount: number }) {
   if (failCount > 0) {
     return (
       <div
-        className="flex h-5 w-5 shrink-0 items-center justify-center rounded-md bg-[#EB5757] text-xs font-bold text-chrome"
+        className="flex h-5 w-5 shrink-0 items-center justify-center rounded-md bg-[#F02D5E] text-xs font-bold text-chrome"
```

Comment:
Would be nice to remove this hard-coded inline color as well since you removed some of the others